src/awala.ts

```diff
@@ -33,3 +33,35 @@ export async function createMemberBundleRequest(
     didSucceed: true,
   };
 }
+
+export async function postToAwala(
+  data: string,
```

Comment:
What type should data be? Should we allow anything and then stringify or just a string?